lineman/app/components/thread_page/thread_page.haml

```diff
@@ -52,3 +52,5 @@
 
     %activity_card{discussion: 'threadPage.discussion'}
     %comment_form{comment: 'threadPage.comment'}
+
+  %thread_lintel
```

Comment:
It's because you're moving it inside the if() block for the thread page, which means that it isn't there when the 'currentComponent' event which tells it that we're viewing the discussion page comes through.

I'll have a further poke a little later on; should be fixable.